src/main/java/com/bgsoftware/superiorskyblock/island/SIslandPreview.java

```diff
@@ -24,11 +24,11 @@ public SIslandPreview(SuperiorPlayer superiorPlayer, Location previewLocation, S
         this.islandName = islandName;
 
         PlayerChat.listen(superiorPlayer.asPlayer(), message -> {
-            if(message.equalsIgnoreCase("CONFIRM")){
+            if(message.equalsIgnoreCase(Locale.ISLAND_PREVIEW_CONFIRM_TEXT.toString())){
```

Comment:
This should be changed to:
`if(message.equalsIgnoreCase(Locale.ISLAND_PREVIEW_CONFIRM_TEXT.getMessage(<user-locale>)))`

Reply:
I completely forgot that users can edit their lang. It should be ok now.